services/azure-servicebus/src/main/java/com/microsoft/windowsazure/services/servicebus/models/ListEventHubsOptions.java

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * Copyright (c) Microsoft and contributors.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.microsoft.windowsazure.services.servicebus.models;
+
+public class ListEventHubsOptions extends AbstractListOptions<ListEventHubsOptions> {
```

Comment:
The current structure of requests follows this pattern and I think it would be more confusing for one model to differ on this.